peps/pep-9999.rst

```diff
@@ -26,18 +25,18 @@ Motivation
 ==========
 
 Packages on PyPI often require build-time and runtime dependencies that are not
-present on PyPI. :doc:`PEP 725 <./pep-0725.rst>`` introduced metadata to express
+present on PyPI. :doc:`pep-0725` introduced metadata to express
 such dependencies. In order to make use of concrete external dependency metadata for
-a Python package, it is necessary to map given dependency identifiers to names used
-by another package manager. The same is true for PyPI package names.
+a Python package, it is necessary to map given dependency identifiers to the names
+used in other ecosystems. The same is true for PyPI package names.
```

Comment:
> The same is true for PyPI package names.

That isn't all that clear actually (my fault I'm sure). Can you read my mind here?